lib/auth/user_agent.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Auth.UserAgent do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Auth.Repo
+
+  schema "user_agents" do
+    field :user_agent, :string
+    has_many :login_logs, Auth.LoginLog
```

Comment:
do we _need_ to have the `has_many` association?

Reply:
My thought was to use Ecto to manage the associations but as mention the person association, we might not want to link the log table directly to the auth data